src/Microsoft.ML.FastTree/FastTreeStatic.cs

```diff
@@ -105,6 +105,48 @@ public static class FastTreeStatic
             return rec.Output;
         }
 
+        /// <summary>
+        /// FastTree <see cref="RankerContext"/>.
+        /// </summary>
+        /// <param name="ctx">The <see cref="RegressionContext"/>.</param>
+        /// <param name="label">The label column.</param>
+        /// <param name="features">The features colum.</param>
+        /// <param name="groupId">The name of the groupId column.</param>
+        /// <param name="weights">The weights column.</param>
```

Comment:
>weights [](start = 38, length = 7)

Maybe while you're at it, "the *optional* weights column" could make clear it's just fine to leave it `null`... though maybe people form this expectation already given that its default value is `null`. I don't know.